pkg/workers/document/document_worker_test.go

```diff
@@ -61,7 +61,7 @@ var _ = Describe("Document resolving", func() {
 			node := &manifest.Node{
 				FileType: manifest.FileType{
 					File:        "node",
-					MultiSource: []string{"https://github.com/gardener/docforge/blob/master/target.md", "https://github.com/gardener/docforge/blob/master/target2.md"},
+					MultiSource: []string{"https://github.com/gardener/docforge/blob/master/target.md", "https://github.com/gardener/docforge/blob/master/target2.md", "https://github.com/gardener/docforge/blob/master/target3.html"},
```

Comment:
Can you change the `It` to "returns correct multisource content from md and html files"